cli/main.ts

```diff
@@ -2,12 +2,22 @@ import { parseCommands } from "./command-parser.ts";
 import { displayHelpCommand } from "./commands/help.ts";
 import { runTasksCommand } from "./commands/tasks.ts";
 import { upgradeCommand } from "./commands/upgrade.ts";
+import { listCommand } from "./commands/list.ts";
 
 const rootCwd = Deno.cwd();
 const { help, command, tasks } = parseCommands(Deno.args);
 
-if (!help && command === "upgrade") {
-  await upgradeCommand();
+if (!help) {
+  switch (command) {
+    case "upgrade": {
+      await upgradeCommand();
+      break;
+    }
+    case "list": {
+      await listCommand(rootCwd);
+      break;
+    }
+  }
   Deno.exit(0);
```

Comment:
issue: This line now prevents any task from running.  For instance, running, `deno task spino test:ci` doesn't do anything.  Removing the early exit, enables that. Ideally, there would be tests to catch this.  I'll add an issue to add some tests.
```suggestion
```